pkgs/native_assets_cli/lib/src/api/code_asset.dart

```diff
@@ -45,25 +45,26 @@ part of 'asset.dart';
 /// "manually", the Dart or Flutter SDK will take care of copying the asset
 /// [file] from its specified location on the current system into the
 /// application bundle.
-abstract final class NativeCodeAsset implements Asset {
+final class CodeAsset implements Asset {
```

Comment:
You moved the other classes that you unified outside the api/ dir. Please move these too, or move the others back into api/.

Reply:
Right now this file is still `part of "src/api/asset.dart";` 

They will move out of `api/` in a follow-up CL, but not into the same places as the other classes - i.e. not directly under `lib/src` (because the concept of `code asset` - like any other asset - is optional and not part of the core protocol).

Though I can temporarily move it under `lib/src`. Done.